ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -103,11 +135,32 @@ export default class StatusDropdown extends React.Component {
       revisionHref: repoModel.getPushLogHref(alertSummary.revision),
       alertHref: `${window.location.origin}/perfherder/alerts?id=${alertSummary.id}`,
       alertSummary: getTextualSummary(filteredAlerts, alertSummary),
+      resultsLink: '',
+      prevResultsLink: '',
     };
 
+    let resultsLink = '';
+    let prevResultsLink = '';
+    if (templateArgs.framework === 'browsertime') {
+      const taskId = await this.getTaskId(alertSummary.push_id);
```

Comment:
It's possible this link creation should be moved in the code that prepares the `alertSummary`.

Reply:
The link creation will be moved wherever is most suitable.